src/components/_modals/DepositModal/SommelierTab.tsx

```diff
@@ -151,8 +153,11 @@ export const SommelierTab: VFC<DepositModalProps> = ({
   })
 
   const { data: selectedTokenBalance } = useBalance({
-    addressOrName: address,
-    token: selectedToken?.address,
+    address: address,
+    token: getAddress(
```

Comment:
why do we need to warp with `getAddress` ? because typing?

Reply:
Fixes a type error